backend/locale/en.po

```diff
@@ -186,125 +186,117 @@ msgstr "I prefer a distribution which is supported by game publishers."
 
 
 msgid "computer-knowledge-question"
-msgstr "How would I rate my knowledge using my computer?"
+msgstr "How would you rate your proficiency using a computer?"
 
 msgid "computer-knowledge-answer-beginner"
-msgstr "I often need help by other users troubleshooting problems with my computer"
+msgstr "I often need help from others to fix problems with my computer"
 
 msgid "computer-knowledge-answer-advanced"
-msgstr "I am already able to solve some problems with my computer by myself"
+msgstr "I am already able to solve some computer problems by myself"
 
 msgid "computer-knowledge-answer-professional"
-msgstr "I can troubleshoot problems with my computer by myself"
+msgstr "I can troubleshoot most or all computer problems by myself"
 
 
 msgid "linux-knowledge-question"
-msgstr "How would I rate my knowledge with Linux itself?"
+msgstr "How would you rate your knowledge of Linux itself?"
 
 msgid "linux-knowledge-answer-beginner"
-msgstr "I do not have any or just a little knowledge about Linux"
+msgstr "I do not have any or just little knowledge about Linux"
```

Comment:
I'd have suggested "I have little or no knowledge about Linux" as more idiomatic English. I can make a PR if you guys agree.

Reply:
You're right, that sounds much better :)

Reply:
Cool! Submitted the change in #73.